sdk/dev-env/lib/dade-dump-profile

```diff
@@ -150,4 +150,5 @@ echo "# Used when running Bazel commands with --config=oracle"
 echo "export ORACLE_USERNAME=system"
 echo "export ORACLE_PWD=hunter2"
 echo "export ORACLE_PORT=1521"
-
+# disable golang proxy
+echo "export GOPROXY=direct"
```

Comment:
Could we comment as to why we disable to GOPROXY? We've had issues with being rate-limited by Github in the past, so if we need to start using a proxy again we may need to know why we're hitting Github directly here.